config/redirects.yml

```
@@ -266,3 +266,7 @@
 "/messages/code-snippets/client-library": "/messages/code-snippets/server-sdk"
 "/dispatch/code-snippets/client-library": "/dispatch/code-snippets/server-sdk"
 "/numbers/code-snippets/buy-a-number": "/numbers/code-snippets/buy"
+"/client-sdk/in-app-voice/code-snippets/reject-call": “/client-sdk/in-app-voice/getting-started/app-to-app-call/ios”
+"/client-sdk/tutorials/in-app-messaging/client-sdk/ip-messaging/add-users-to-conversation": "/client-sdk/tutorials/in-app-messaging/client-sdk/in-app-messaging/add-users-to-conversation"
+"voice/guides/connecting-voice-calls-to-amazon-lex-bots": "/voice/voice-api/guides/connecting-voice-calls-to-amazon-lex-bots"
```

@niati there's a missing `/` at the beginning of this route
```suggestion
"/voice/guides/connecting-voice-calls-to-amazon-lex-bots": "/voice/voice-api/guides/connecting-voice-calls-to-amazon-lex-bots"
```